extras/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ function(rwkv_add_extra source)
 	get_filename_component(EXTRA_TARGET ${source} NAME_WE)
 	add_executable(rwkv_${EXTRA_TARGET} ${source})
 	target_link_libraries(rwkv_${EXTRA_TARGET} PRIVATE ggml rwkv)
+	if (RWKV_STATIC)
+		get_target_property(target_LINK_OPTIONS rwkv_${EXTRA_TARGET} LINK_OPTIONS)
```

Comment:
Why there is `rwkv_` prefix in `rwkv_${EXTRA_TARGET}`, but no such prefix in [tests/CMakeLists.txt](https://github.com/saharNooby/rwkv.cpp/pull/98/files#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0d)?

Reply:
I have no idea why there is prefix. Its looks like you defined it in CMakeLists :D